docs/jobs/openings/dev_rel.md

```diff
@@ -1,34 +1,39 @@
 ## Developer Relations opening at Aragon
 
-We are looking for a developer relations to help third party developers create apps on top of the [Aragon dapp](https://github.com/aragon/aragon) and [aragonOS](http://wiki.aragon.one/documentation/aragonOS/).
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world's worst problems.
+
+We're looking for a Developer Relations to help third party developers create apps on top of the [Aragon dapp](https://github.com/aragon/aragon) and [aragonOS](http://wiki.aragon.one/documentation/aragonOS/). Our Dev Rels must be someone passionate in helping others understand technical subjects through technical writing, networking and educating people. That means that our ideal Dev Rels is not only a legit developer but also an Aragon Technical Evangelist.
 
 ### Responsibilities
 
-- Nurture the Aragon dev ecosystem
-- Attend and organize hackathons, helping devs get excited and started developing on Aragon.
-- Developer tutorials and documentation
-- Help external contributors
-- Find interesting projects that might be eligible for grants
+- Write technical documentation and tutorials for developers
+- Nurture and support the Aragon dev ecosystem
+- Attend and organize hackathons, helping devs get excited and started developing on Aragon
+- Support external contributors
+- Source, screen and select interesting projects that might be eligible for grants
```

Comment:
Should we specify that this means Nest grants? Might help people learn more about Nest and why they should do this work for eligible grants.